src/cmocka/rnp_tests_generatekey.c

```diff
@@ -491,13 +491,26 @@ rnpkeys_generatekey_verifykeyHomeDirNoPermission(void **state)
     rnp_end(&rnp);
 }
 
-static void
-ask_expert_details(rnp_t *ctx, const char *rsp, size_t rsp_len)
+static bool
+ask_expert_details(rnp_t *ctx, rnp_cfg_t *ops, const char *rsp, size_t rsp_len)
 {
+    /* Run tests*/
+    bool      ret = true;
+    rnp_cfg_t cfg = {0};
+    if (setup_rnp_common(ctx, GPG_KEY_STORE, NULL, NULL) != RNP_OK) {
+        return false;
```

Comment:
I feel confused baceuse we have:
 - `RNP_OK`/`RNP_FAIL`;
 - `true`/`false`;
 - `0`/`1`/`-1`.

I didn't say that it is a error but I feel that we should find a standard way because right now it something strange.